string cube_colour
float64 normalisedCoordinateX
float64 normalisedCoordinateY
float64 area
float64 width
float64 height
float64 distance
